services/web/project/static/swagger.yml

```
@@ -0,0 +1,275 @@
+        - contacts
+      summary: "Create a user"
+      requestBody:
+          description: "User to create"
+          content:
+            application/json:
+              schema:
+                x-body-name: "user" 
+                $ref: "#/components/schemas/User"       
+      responses:
+        '200':
+          description: "Successfully created user" 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)